### ISSUE TITLE
Following Quick Start Guide generates error at step two

### ISSUE THREAD

**Issue by supersexy** [ubuntu/microk8s#828]
The second command of the [Quick Start Guide](https://microk8s.io/docs/) generates an error:
```
vagrant@microk8s:~$ microk8s.status --wait-ready
Insufficient permissions to access MicroK8s.
You can either try again with sudo or add the user vagrant to the 'microk8s' group:

    sudo usermod -a -G microk8s vagrant

The new group will be available on the user's next login.

```

It is an obvious one, of course, just a small thing, but as a quick starter I now have to ask myself:
- What are the implications if I add the current user to the microk8s group?
- Will everything still work? 
- How can I be sure that adding the user to the group is supported by microk8s?
- Or do I have to use microk8s with sudo _all the time_?
- Why is information about such an important thing missing?
- Why is the second command in a documentation of one of the leaders in Linuxland generating errors? 
- Will there be even more errors?
- Why is nobody testing this?
- Will I have lots of problems using a K8S distribution that generates errors on the _second_ command found in the quick start? 
- Is this project dead?
- Should I dig deep into the implications of K8S and system groups?
- Is this a quick and easy tool that I can "just use"?
- Why is this stopping my workflow?
- Should I build my own K8S distribution?
- What do I have to think of a company that publishes quick guides that fail? 
- Where went the software quality level of the good old days?
- Why do we have to spend so much time with computers that constantly produce errors?
- Why is there so much darkness in the universe?
- Why do we all have to die? 

You get it... of course I still love you, but please do not do such things. Step two in a quick start guide should be checked by at least three persons before publishing it. Thank you!

**Comment by ktsakalozos** [comment]
You are right @supersexy. We should be more precise in our docs. Unfortunate all users that gave the +1 to the docs were already in the microk8s group so we missed this error.

@evilnick I think we should add the `sudo` and point the users to a doc discussing the microk8s group. What do you say?

**Comment by usamaB** [comment]
when I close a terminal session and try to run kubectl command I get the same insufficient permission error again. the group isn't persisting through sessions.

**Comment by Nocturem** [comment]
Seconding this I ran straight into this issue and the first result on googling takes you off to [microk8s.kubectl requires sudo](https://github.com/ubuntu/microk8s/issues/606)

its not ideal to run into this on the second item in a guide on https://microk8s.io/
perhaps while the project is still actively developing it may be prudent to redirect users to the github readme as that looks to be accurate. 

Interestingly being a member of the microk8s group does not actually seem to fix the problem at least not on Mint 19.3

**Comment by galgalesh** [comment]
Bump, this is still an issue. Almost every new user will start with this guide and thus experience this.
